reproduce-figures/utils.R

```diff
@@ -0,0 +1,69 @@
+# This file contains functions used in prepare-scpca-portal-data.R to process consensus cell types.
+
+
+
+
+
+#' Prepare and export TSV of consensus cell types
+#'
+#' @param metadata Metadata slot of an SCE object 
+#' @param sce_coldata Coldata slot of an SCE object 
+#' @param output_tsv Path to output TSV file
+#'
+prepare_consensus_tsv <- function(sce_metadata, sce_coldata, output_tsv){
```

Comment:
```suggestion
prepare_consensus_tsv <- function(sce, output_tsv){
```
I think it would be more straight forward to just read in the sce and grab the col and metadata in the function.